modules/installation-configuration-parameters.adoc

```diff
@@ -3056,6 +3089,23 @@ Additional Nutanix configuration parameters are described in the following table
 |The virtual IP (VIP) address that you configured for control plane API access.
 |IP address
 
+|platform:
+  nutanix:
+    failureDomains:
+    - name:
+      prismElement:
+        name:
+        uuid:
+      subnetUUIDs:
+      -
+a|By default, the installation program installs cluster machines to a single Prism Element instance. You can specify additional Prism Element instances for fault tolerance, and then apply them to:
```

Comment:
Maybe being picky, but you called it a Prism Element (without the _instance_) [above](https://github.com/openshift/openshift-docs/pull/69618/files#diff-226446e7ce5ccad467271e158f45f2d078bc87cfa65bf9023243bc60657cdc83R9).

```suggestion
a|By default, the installation program installs cluster machines to a single Prism Element. You can specify additional Prism Elements for fault tolerance, and then apply them to:
```

Reply:
Truthfully @mburke5678 I think we are OK here. While the line you reference does not have _instance_, the line below it does.